scheduler_plugins/plugin_template/additional_cluster_infrastructure.cfn.yaml

```diff
@@ -0,0 +1,66 @@
+# Available Jinja variables:
```

Comment:
what about using for this file and the followings the paths
`/scheduler_plugins/slurm/` and `/scheduler_plugins/template/`?

In this way it's aligned with the definition of the template, e.g. 
```
ClusterInfrastructure:
  CloudFormation:
    Template: s3://<BUCKET>/scheduler_plugins/slurm/slurm_plugin_infrastructure.cfn.yaml
```

Reply:
I'd keep them in the root for now being them the main entrypoints of the plugin

Reply:
ok I had misunderstood the comment. Renamed slurm_plugin root directory to slurm